app/src/main/java/fr/free/nrw/commons/upload/LanguagesAdapter.kt

```diff
@@ -26,6 +26,16 @@ class LanguagesAdapter constructor(
     private val selectedLanguages: HashMap<*, String>
 ) : ArrayAdapter<String?>(context, R.layout.row_item_languages_spinner) {
 
+    companion object {
+        /**
+         * Represents the default index for the language list. By default, this index corresponds to the
+         * English language. This serves as a fallback when the user's system language is not present in
+         * the language_list.xml. Though this default can be changed by the user, it does not affect other
+         * functionalities of the application. Fix bug issue 5338
```

Comment:
tiny change: Would you mind changing "Fix" to "Fixes" ?

Reply:
ok

Reply:
Hi, I fix that, can you make approval for it? Thanks!